Integrations/integration-VMRay.yml

```diff
@@ -0,0 +1,106 @@
+description: Ransomware analysis sandboxing
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API-Key
+  name: api_key
+  defaultvalue: ""
+  type: 4
+  required: false
+script:
+  script: |+
+    import base64
+    import datetime
+    import os.path
+    import requests
+    import urlparse
+    import time
+    from vmray.rest_api import VMRayRESTAPI, VMRayRESTAPIError
+
+
+    # Get mandantory params
+    # You can use demisto.params()[paramName] or params.paramName to get a specific params.
+    # Params are of the type given in the integration page creation.
+    api_key = demisto.params()['api_key']
+    server = demisto.params()['server']
+
+    api = VMRayRESTAPI(server, api_key)
+
+    url_dict = {
+        "get_results": "/rest/analysis/sample/%s",
+        "get_job_sample": "/rest/job/sample/%s"
+    }
+
+    def submit_sample(api, file):
+        # Submit sample
+        params = {'sample_file': file}
+        data = api.call("POST", "/rest/sample/submit", params)
+        demisto.results(json.dumps(data, indent=2))
+
+    def test_module(api):
+        try:
+            api.call("GET", "/rest/analysis/?_limit=1")
```

Comment:
I think it might be better to try and except to wrap the error in a more proper message.